misc/dbt-materialize/dbt/include/materialize/macros/apply_grants.sql

```diff
@@ -0,0 +1,33 @@
+-- Copyright 2020 Josh Wills. All rights reserved.
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{%- macro quote_grantees(grantees) -%}
```

Comment:
Seems like this function should exist as util within dbt but couldn't find it.

Reply:
Hum. Is the purpose here to handle grants with multiple role assignments? If so, it looks like [`get_dcl_statement_list`](https://github.com/dbt-labs/dbt-core/blob/eac13e3bd38b7f29401444ca77ee243ec4d8f29d/core/dbt/include/global_project/macros/adapters/apply_grants.sql#L92) might be it?

Reply:
This is just to correctly quote the role names. As @joacoc noticed a role name like `test@materialize.com` would fail